rs/rosetta-api/icrc1/ledger/tests/tests.rs

```diff
@@ -402,7 +398,7 @@ fn icrc1_test_upgrade_serialization() {
     ic_icrc1_ledger_sm_tests::test_upgrade_serialization(
         ledger_mainnet_wasm(),
         ledger_wasm(),
-        Some(ledger_wasm_nextmigrationversionmemorymanager()),
+        None,
```

Comment:
Shouldn't this be cleaned up, too?

Reply:
It will be used by the ICP ledger tests.